tests/test_cmdi_parser.py

```diff
@@ -70,5 +70,142 @@ def test_get_issued_not_empty(single_record):
 def test_json_converter(single_record):
     record = MSRecordParser(single_record)
     result = record.json_converter()
-    expected_result = json.dumps({"persistent_identifier": "urn.fi/urn:nbn:fi:lb-2017021609", "title": {"en": "Silva Kiuru's Time Expressions Corpus", "fi": "Silva Kiurun ajanilmausaineisto"}, "description": {"en": "This corpus of time expressions has been compiled from literary works, translations, dialect texts as well as other texts. Format: word documents.", "fi": "T\u00e4m\u00e4 suomen kielen ajanilmauksia k\u00e4sitt\u00e4v\u00e4 aineisto on koottu kaunokirjallisten alkuper\u00e4isteosten, k\u00e4\u00e4nn\u00f6sten, murreaineistojen ja muiden tekstien pohjalta."}, "modified": "2017-02-15T00:00:00.000000Z", "issued": "2017-02-15T00:00:00.000000Z"})
-    assert result == expected_result
\ No newline at end of file
+    expected_result = json.dumps(
+        {
+    "persistent_identifier": "urn.fi/urn:nbn:fi:lb-2017021609",
+    "title": {
+        "en": "Silva Kiuru's Time Expressions Corpus",
+        "fi": "Silva Kiurun ajanilmausaineisto"
+    },
+    "description": {
+        "en": "This corpus of time expressions has been compiled from literary works, translations, dialect texts as well as other texts. Format: word documents.",
+        "fi": "T\u00e4m\u00e4 suomen kielen ajanilmauksia k\u00e4sitt\u00e4v\u00e4 aineisto on koottu kaunokirjallisten alkuper\u00e4isteosten, k\u00e4\u00e4nn\u00f6sten, murreaineistojen ja muiden tekstien pohjalta."
+    },
+    "modified": "2017-02-15T00:00:00.000000Z",
+    "issued": "2017-02-15T00:00:00.000000Z",
+    "access_rights": {
+        "license": [
+            {
+                "url": "http://uri.suomi.fi/codelist/fairdata/license/code/undernegotiation"
+            }
+        ],
+        "access_type": {
+            "access_type": "http://uri.suomi.fi/codelist/fairdata/access_type/code/open"
+        }
+    }
+}
+    )
+    assert result == expected_result
```

Comment:
This test fails:
```
>       assert result == expected_result
E       assert '{"data_catal...code/open"}}}' == '{"persistent...code/open"}}}'
```

Reply:
The failing test was missing the hard coded dummy data I added at to the json_converter(). The test passes now.

I left a comment about how to avoid the hanging for now. We'll need to fix it later.